distribution/lib/Standard/Base/0.0.0-dev/src/Polyglot/Proxy_Polyglot_Array.enso

```diff
@@ -0,0 +1,23 @@
+from Standard.Base import Polyglot
+
+## Advanced
+
+    Wrapper for Polyglot Arrays
+type Proxy_Polyglot_Array
+
+    type Proxy_Polyglot_Array arr
+
+    ## Returns the number of elements stored in this Polyglot Array.
+
+       > Example
+         Checking the length of a vector.
+
+             [1, 2, 3, 4].length
```

Comment:
Is `[1, 2, 3, 4]` allocating an array or a vector? Is the example checking length of an `Array` at all?

Reply:
Nope this would be just checking a Vector length.

Reply:
My bad, this is now fixed